src/elixirServer.ts

```diff
@@ -116,6 +116,23 @@ export class ElixirServer {
         this.sendRequest('DEFL', command, resultCb);
     }
 
+    getDocumentation(document: vscode.TextDocument, position: vscode.Position, callback: Function): void {
+        const wordAtPosition = document.getWordRangeAtPosition(position);
+        const word = document.getText(wordAtPosition);
+        if (word.indexOf('\n') >= 0) {
+            console.error('[vscode-elixir] got whole file as word');
+            callback([]);
+            return;
+        }
+        const command: string = `DOCL { "${word}", "${document.fileName}", ${position.line + 1} }\n`;
+        const resultCb = (result: string) => {
+            console.log(result);
```

Comment:
Why do we `console.log()` here? Is this for debugging? Could we maybe make it so that this doesn't fire in production?